test/gpu/native/studies/chop/chplGPU.skipif

```diff
@@ -11,11 +11,14 @@ CHOP_BRANCH=main
 CHOP_URL=${CHOP_URL:-https://github.com/tcarneirop/ChOp.git}
 CHOP_BRANCH=${CHOP_BRANCH:-main}
 
-# Clone ChOp, skipif clone failed, add extra output to fail nightly job
+# Clone ChOp, skipif clone failed. Timeouts are frequent enough we don't want
+# to cause a failure.  It would be nice to add extra output to the log to
+# explain why it skipped but any output beyond "True" or "False" (even if sent
+# to stderr) will cause the skipif itself to fail.
 rm -rf ChOp
 if ! git clone ${CHOP_URL} --branch=${CHOP_BRANCH} --depth=1 2>gitClone.out; then
-  echo "git clone failed; output:" >&2
-  cat gitClone.out >&2
```

Comment:
Maybe we just drop the output into a custom file of sorts and not `cat` that file. If there's a persistent git failure, we can always find that file in the jenkins workspace.

Reply:
Ugh, I was reading too fast. Ok, yeah we have that already, that should be sufficient for more debugging if needed.